load.bzl

```diff
@@ -62,10 +62,10 @@ def repositories():
     if not native.existing_rule("io_bazel_rules_go"):
         http_archive(
             name = "io_bazel_rules_go",
-            sha256 = "7904dbecbaffd068651916dce77ff3437679f9d20e1a7956bff43826e7645fcc",
+            sha256 = "dbf5a9ef855684f84cac2e7ae7886c5a001d4f66ae23f6904da0faaaef0d61fc",
             urls = [
-                "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.25.1/rules_go-v0.25.1.tar.gz",
-                "https://github.com/bazelbuild/rules_go/releases/download/v0.25.1/rules_go-v0.25.1.tar.gz",
+                "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.24.11/rules_go-v0.24.11.tar.gz",
+                "https://github.com/bazelbuild/rules_go/releases/download/v0.24.11/rules_go-v0.24.11.tar.gz",
```

Comment:
You can drop this change:
1. It's a downgrade.
2. `rules_go` v0.25.0 allows fetching a newer go version without needing a hardcoded update: https://github.com/bazelbuild/rules_go/releases/tag/v0.25.0

Reply:
i saw that was a downgrade, but did not see any 0.25 saying the go update.
thanks for the clarification